test/Microsoft.ML.Tests/Scenarios/Api/AspirationalExamples.cs

```diff
@@ -61,4 +61,89 @@ public void FirstExperienceWithML()
             });
         }
     }
+
+    public class GithubClassification
+    {
+        public void ClassifyGithubIssues()
+        {
+            var env = new TlcEnvironment(new SysRandom(0), verbose: true);
+
+            string dataPath = "corefx-issues-train.tsv";
+
+            // Create reader with specific schema. 
+            // string :ID, string: Area, string:Title, string:Description
+            var reader = TextLoader.CreateReader(env, ctx =>
+                                          (area: ctx.LoadText(1),
+                                            title: ctx.LoadText(2),
+                                            description: ctx.LoadText(3),
+                                            dataSource,
+                                            useHeader: true));
+
+            var data = reader.Read(dataPath).AsDynamic;
+
+            var schema = textData.Schema;
+
+            // Verify that the columns are there. 
+            // There ought to be at least one column corresponding to the identifiers in the tuple.
+            CheckSchemaHasColumn(schema, "area", out int areaIdx);
+            CheckSchemaHasColumn(schema, "title", out int titleIdx);
+            CheckSchemaHasColumn(schema, "description", out int descriptionIdx);
+
+            // Next verify they have the expected types.
+            Assert.Equal(TextType.Instance, schema.GetColumnType(areaIdx));
+            Assert.Equal(TextType.Instance, schema.GetColumnType(titleIdx));
+            Assert.Equal(TextType.Instance, schema.GetColumnType(descriptionIdx));
+
+            // sefilipi: Do we need to inspect the data, as in Tom's example?	
+
+            var estimator = Estimator.MakeNew(data)
```

Comment:
>data [](start = 46, length = 4)

if it's the `Dynamic` thing, you won't be able to spin off a pigsty estimator out of it. It should be non-dynamic